models/mlp/model.py

```diff
@@ -6,10 +6,8 @@ class MLP(nn.Module):
 
     def __init__(self, config):
         super().__init__()
-        dataset = config.dataset
-        target_class = config.target_class
         self.dropout = nn.Dropout(config.dropout)
-        self.fc1 = nn.Linear(dataset.VOCAB_SIZE, target_class)
+        self.fc1 = nn.Linear(config.vocab_size, config.num_labels)
```

Comment:
Should probably rename MLP to LR in the future.

Reply:
I think we should rename it to perceptron. Since we use sigmoid activation for multilabel, I feel that calling it LR would be misleading.